android-reactive-location/src/main/java/pl/charmas/android/reactivelocation/observables/location/LocationUpdatesObservable.java

```diff
@@ -38,8 +38,8 @@ protected void onGoogleApiClientReady(GoogleApiClient apiClient, final Observer<
     protected void onUnsubscribed(GoogleApiClient locationClient) {
         if (locationClient.isConnected()) {
             LocationServices.FusedLocationApi.removeLocationUpdates(locationClient, listener);
+            listener.unsubscribe();
```

Comment:
I think we should add `@Nullable` to `listener` and always check non-null before calling `unsubscribe()`.